osu.Game/Localisation/DebugSettingsStrings.cs

```diff
@@ -49,6 +49,16 @@ public static class DebugSettingsStrings
         /// </summary>
         public static LocalisableString CompactRealm => new TranslatableString(getKey(@"compact_realm"), @"Compact realm");
 
+        /// <summary>
+        /// "Block realm"
+        /// </summary>
+        public static LocalisableString BlockRealm => new TranslatableString(getKey(@"block_realm"), @"Block realm");
+
+        /// <summary>
+        /// "Unblock realm"
+        /// </summary>
+        public static LocalisableString UnblockRealm => new TranslatableString(getKey(@"unblock_realm"), @"Unblock realm");
```

Comment:
These shouldn't be localised, and the "compact realm" should probably be removed. These are debug level settings and I don't want to bother translators with figuring out what they mean.

Reply:
Okay, I will remove it. "Compact realm" should also be non-localizable?

Reply:
I think so, yeah. Users shouldn't be using these functions without guidance.

Reply:
I understand. Thank you!